Makefile

```diff
@@ -17,7 +17,8 @@
 CGO_ENABLED=0
 GOOS=linux
 TEST_IMAGES=$(shell find -L ./test/test_images -mindepth 1 -maxdepth 1 -type d)
-KO_DOCKER_REPO=${DOCKER_REPO_OVERRIDE}
+TEST=
+DOCKER_REPO_OVERRIDE=
```

Comment:
Hi @dsimansk, it seems that this way the DOCKER_REPO_OVERRIDE is going to be always an empty string?

Reply:
Actually, it's inspired by Serving's [Makefile](https://github.com/openshift/knative-serving/blob/master/Makefile#L7). I assumed it's intentional to avoid accidental overrides from env. 
Then it's meant to be executed as:
```
make DOCKER_REPO_OVERRIDE=<repository> test-images
```
If it doesn't make sense here I don't mind changing to favour `DOCKER_REPO_OVERRIDE` env var.

Reply:
Ah ok, I had env var exported and wondered why it doesn't work. It works the way you mentioned, so I am fine with leaving it as is. Thanks!